README.md

```diff
@@ -31,7 +36,7 @@ On a local machine:
 3. Set the location of the weather database in the Main.R script of the R_program folder (where it says set database location), along with the name of the weather database (where it says Provide the name of the database in quotes).
```

Comment:
Maybe the `Main.R` script could be made more user friendly by providing a default for `db_loc` that corresponds to step 2?
```
#Set database and inputs location
db_loc<-"../inputs"
```

Reply:
@dschlaep sure great suggestion!

Reply:
oh, no, my suggestion will fail on line 602 of `Main.R`; this should be rather something like 
```
    db_loc <- file.path(source.dir, "..", "inputs”)
```

Reply:
Here, source.dir is the path to the individual R_program folder for the site. The inputs folder is embedded in the rSFSTEP2 folder, not the R_program folder, so 

`db_loc <- file.path(source.dir, "..", "inputs”)
`

will not work. I will provide an example of what the db_loc path should look like. I don't think it is too difficult to ask users to set this one file path. Every other file path is automated. 

Thoughts?

Reply:
Addressed by https://github.com/DrylandEcology/rSFSTEP2/commit/9f2637ea7f2631e32ce9b3961ce935d748a7396c